packages/analyzer/src/features/analyse-phase/creators/handlers.js

```diff
@@ -139,9 +139,15 @@ export function createClassDeclarationMixin(name, moduleDoc) {
 
 /**
  * Handles mixins and superclass
+ + * @param {import('custom-elements-manifest/schema').ClassDeclaration} classTemplate
+ + * @param {import('custom-elements-manifest/schema').Module} moduleDoc
+ + * @param {import('typescript').ClassDeclaration} node
```

Comment:
Types are not needed. I made a conscious choice to remove TS from the codebase, for now I'd like to avoid using types and avoid _some_ code having types and _some_ code not having types

Reply:
done